specification/web/resource-manager/Microsoft.DomainRegistration/stable/2021-03-01/Domains.json

```diff
@@ -708,6 +708,56 @@
           }
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.DomainRegistration/domains/{domainName}/transferOut": {
+      "put": {
+        "tags": [
+          "Domains"
+        ],
+        "summary": "Transfer out domain to another registrar",
+        "operationId": "Domains_TransferOut",
+        "parameters": [
+          {
+            "$ref": "#/parameters/resourceGroupNameParameter"
+          },
+          {
+            "name": "domainName",
+            "in": "path",
+            "description": "Name of domain.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully returned transfer code to transfer domain to another registrar.",
+            "schema": {
+              "$ref": "#/definitions/Domain"
+            }
+          },
+          "400": {
+            "description": "Domain does not exist in Azure database",
+            "x-ms-error-response": true
+          },
```

Comment:
I think it still need a schema (like the on in `default`)?

Reply:
A lot of the other "400" error in Microsoft.Web RP don't have schema so I didn't add to remain consistent